android/src/com/mapswithme/maps/downloader/MigrationFragment.java

```diff
@@ -128,23 +128,28 @@ public void onClick(View v)
       }
     });
 
-    MigrationController.get().attach(this);
-
     Statistics.INSTANCE.trackEvent(Statistics.EventName.DOWNLOADER_MIGRATION_DIALOG_SEEN);
   }
 
   @Override
-  public void onDestroyView()
+  public void onResume()
```

Comment:
А для чего вообще нужно сохранять ссылку на фрагмент?

> On Apr 5, 2016, at 14:45, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/downloader/MigrationFragment.java <https://github.com/mapsme/omim/pull/2736#discussion_r58523730>:
> 
> >      Statistics.INSTANCE.trackEvent(Statistics.EventName.DOWNLOADER_MIGRATION_DIALOG_SEEN);
> >    }
> >  
> >    @Override
> > -  public void onDestroyView()
> > +  public void onResume()
> В отличие от onPause() onStop() зовётся, когда фрагмент не виден.
> attach/detach сохраняют/удаляют ссылку на фрагмент, restore восстанавливает состояние UI.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2736/files/9a33ba337e717266a346a3a58375ec71f2bca26e#r58523730>